appveyor.yml

```diff
@@ -0,0 +1,45 @@
+version: 1.0.{build}
+
+clone_folder: c:\projects\osqp-eigen
+
+environment:
+  Eigen3_DIR:          C:/Program Files (x86)/Eigen/lib/cmake/eigen3/
+
+os:
+ - Visual Studio 2015
+ - Visual Studio 2017
+
+install:
+  # Check env variables
+  - cmd: echo CMAKE_PREFIX_PATH %CMAKE_PREFIX_PATH%
+  - cmd: echo PATH %PATH%
+build:
+
+build_script:
+  # download and build osqp
+  - cd c:\projects
+  - git clone --recursive https://github.com/oxfordcontrol/osqp.git
+  - cd osqp
+  - md build
+  - cd build
+  - cmake ..
+  - msbuild /m /p:Configuration=Release /p:Platform="Win32" osqp.sln
+  - cmake --build . --config Release
+  - cmake --build . --config Release --target INSTALL
+  # download and install eigen3
+  - cd c:\projects
+  - hg clone https://bitbucket.org/eigen/eigen
+  - cd eigen
+  - hg checkout 3.3-beta2
+  - md build
+  - cd build
+  - cmake ..
+  - cmake --build . --config Release --target INSTALL
+  # compile osqp-eigen
+  - cd c:\projects\osqp-eigen
+  - md build
+  - cd build
+  - cmake .. -DEIGEN3_INCLUDE_DIR="C:/Program Files (x86)/Eigen/include/eigen3"
+  - msbuild /m /p:Configuration=Release /p:Platform="Win32" OsqpEigen.sln
```

Comment:
I guess this line is a duplicate of the following one, so it can be removed.

Reply:
Actually also 
```
  - msbuild /m /p:Configuration=Release /p:Platform="Win32" osqp.sln
  - cmake --build . --config Release
  - cmake --build . --config Release --target INSTALL
```
are duplicated lines. So at the end I can leave only 
```
- cmake --build . --config Release --target INSTALL
```
@traversaro correct me if I'm wrong

Reply:
Indeed!

Reply:
Ok, so I think that there are duplicated line also [here](https://github.com/robotology/idyntree/blob/master/appveyor.yml).

Reply:
Good catch!

Reply:
Done in dbda925